src/Upgrade/Application/Strategy/AbstractStrategy.php

```diff
@@ -53,4 +63,28 @@ public function upgrade(): StepsResponseDto
 
         return $stepsExecutionDto;
     }
+
+    /**
+     * @param \Upgrade\Application\Strategy\StepInterface $step
+     * @param \Upgrade\Application\Dto\StepsResponseDto $stepsResponseDto
+     *
+     * @return \Upgrade\Application\Dto\StepsResponseDto
+     */
+    protected function runWithFixer(StepInterface $step, StepsResponseDto $stepsResponseDto): StepsResponseDto
+    {
+        foreach ($this->fixers as $fixer) {
+            if (!$fixer->isApplicable($stepsResponseDto)) {
+                continue;
+            }
+            $stepsResponseDto = $fixer->run($stepsResponseDto);
+            if ($stepsResponseDto->getIsSuccessful()) {
+                $stepsResponseDto = $step->run($stepsResponseDto);
```

Comment:
Please use `continue` to avoid multiple nesting.

Reply:
where? we break from loop if we fixed or don't have more fixers